modules/default/updatenotification/update_helper.js

```diff
@@ -0,0 +1,162 @@
+const Exec = require("child_process").exec;
+const Spawn = require("child_process").spawn;
+const commandExists = require("command-exists");
+const Log = require("logger");
+
+class Updater {
+	constructor(config, callback) {
```

Comment:
can callback not better be done with an async await promise based solution?

Reply:
I make callbacks for live displaying.
Sure, i can make async promise with all state informations in revolve/reject

I will works about that

Reply:
done